.markdownlint-cli2.cjs

```diff
@@ -0,0 +1,11 @@
+const options = require('@github/markdownlint-github').init({
+  "MD013": false, // Line length
+  "MD041": false, // First line in file should be a top level heading
```

Comment:
I added these based on violations I found in your codebase which seemed okay to me.